Resources/Prototypes/Traits/physical.yml

```diff
@@ -569,14 +555,14 @@
           allowedStates:
           - Alive
           - Critical
-          damageCap: 200
+          damageCap: 400
           damage:
             groups:
-              Brute: -0.07
-              Burn: -0.07
-              Airloss: -0.07
-              Toxin: -0.07
-              Genetic: -0.07
+              Brute: -0.35
+              Burn: -0.35
+              Airloss: -0.25
```

Comment:
Lets replace the Airloss group with maybe -0.07 Bloodloss (individual damage, not a group). The pre-buff bloodloss healing was actually one of its best types to be fair.